tests/test_crawl.py

```diff
@@ -361,27 +351,6 @@ def cb(response):
         assert s["engine.spider.name"] == crawler.spider.name
         assert s["len(engine.scraper.slot.active)"] == "1"
 
-    @defer.inlineCallbacks
-    def test_graceful_crawl_error_handling(self):
```

Comment:
Is this test removed on purpose?

Reply:
It was because I first switched it to async def generator, and that scenario was covered in later changes (in #6715, `test_exception_before_yield`).

But it seems it would make sense to keep a test about this for non-generator `start_requests`.